app/routes/__index/dns-records/index.tsx

```diff
@@ -77,6 +80,29 @@ export const action = async ({ request }: ActionArgs) => {
   }
 };
 
+function mapStatusToErrorText(statusCode: number): string {
+  switch (statusCode) {
+    case 404:
+      return 'Sorry we could not find your dns record';
+    case 400:
+      return 'We got an error processing requested action on your dns record';
+    default:
+      return getErrorMessageFromStatusCode(statusCode);
+  }
+}
+
+export function CatchBoundary() {
+  const caught = useCatch();
+
+  return <SeenErrorLayout result={caught} mapStatusToErrorText={mapStatusToErrorText} />;
+}
+
+export function ErrorBoundary() {
+  return (
+    <UnseenErrorLayout errorText="We got an unexpected error working with your dns records, but don't worry our team is already on it's way to fix it" />
```

Comment:
DNS Record.

Is our team already on its way to fixing this?  That seems hopeful, but also not true :)